jetstream/external_config.py

```diff
@@ -108,9 +210,20 @@ def from_github_repo(cls) -> "ExternalConfigCollection":
 
             external_configs = []
 
+            config_files_to_skip = (
+                "bug-1695015-pref-new-tab-modernized-ux-region-1-release-86-88.toml",
+                "bug-1671484-pref-validation-of-relpreload-performance-impact-release-82-83.toml",
+                "bug-1726656-pref-tab-unloading-nightly-93-94.toml",
+            )  # TODO: temporary, remove once resolved issue with this specific config
+
```

Comment:
this can be removed?
```suggestion
```

Reply:
Unfortunately not, if we want to keep this validation logic:
```
FAILED jetstream/tests/test_external_config.py::TestExternalConfig::test_from_github_repo - jetstream.errors.MetricsConfigurationException: It appears some configuration for specified metrics is missing: {'time_to_first_interaction_preload_ms', 'time_to_first_interaction_no_preloa...
FAILED jetstream/tests/test_metadata.py::test_metadata_from_config - jetstream.errors.MetricsConfigurationException: It appears some configuration for specified metrics is missing: {'time_to_first_interaction_preload_ms', 'time_to_first_interaction_no_preload_ms'}
FAILED jetstream/tests/test_metadata.py::test_metadata_reference_branch - jetstream.errors.MetricsConfigurationException: It appears some configuration for specified metrics is missing: {'time_to_first_interaction_preload_ms', 'time_to_first_interaction_no_preload_ms'}
FAILED jetstream/tests/test_metadata.py::test_metadata_from_config_missing_metadata - jetstream.errors.MetricsConfigurationException: It appears some configuration for specified metrics is missing: {'time_to_first_interaction_preload_ms', 'time_to_first_interaction_no_preload_ms'}
FAILED jetstream/tests/test_metadata.py::test_export_metadata - jetstream.errors.MetricsConfigurationException: It appears some configuration for specified metrics is missing: {'time_to_first_interaction_preload_ms', 'time_to_first_interaction_no_preload_ms'}
```

Reply:
We would have to fix the configs in the repo first. I'm not sure if we can just do that or could this have undesirable effects

Reply:
Fixed via https://github.com/mozilla/jetstream-config/commit/2ce9e92f7f0497466b24a152b6af01559d5348fb

I removed the config skipping here